packages/blade/src/components/Box/BaseBox/types/spacingTypes.ts

```diff
@@ -11,7 +11,9 @@ type SpacingValueType =
   | `min(${string})`
   | `max(${string})`
   | 'auto'
-  | 'none';
+  | 'none'
+  | 'initial'
+  | 'fit-content';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/width#values

can we add `max-content` and `min-content` as well in this PR itself?

Reply:
sure